src/App/App.module.scss

```
@@ -0,0 +1,8 @@
+.App {
+  text-align: center;
+  font-family: 'Roboto', sans-serif;
+}
+.main {
+  margin: auto;
+  width: 100px;
```

лучше использовать width + max-width
```suggestion
  max-width: 100px;
  width: 100%;
```

В таком случае на больших экранах будет максимум 100, на маленьких - 100% от маленького экрана